README.md

```diff
@@ -9,10 +9,10 @@
             - [Service Bus logging](#service-bus-logging)
         - [Transfer parameter reference](#transfer-parameter-reference)
             - [Cobalt task transfer parameters](#cobalt-task-transfer-parameters)
-                - [Source](#source)
-                - [Destination](#destination)
-                - [Message processing steps](#message-processing-steps)
-                - [Parameters](#parameters)
+        - [Source](#source)
+        - [Destination](#destination)
+        - [Message processing steps](#message-processing-steps)
+        - [Parameters](#parameters)
```

Comment:
Ah, are these now moved too high in the TOC as well? Should we just update the structure a bit, e.g. remove the "Cobalt task transfer parameters" level altogheter?